doc/Changelog.md

```diff
@@ -2,6 +2,9 @@
 
 # git master
 
+* [185](https://github.com/HBPVIS/ZeroEQ/pull/185):
+  http::server::getSocketDescriptor() notifies on write after
+  [#184](https://github.com/HBPVIS/ZeroEQ/pull/184).
```

Comment:
no, just putting a link to the pull request that introduced the change of semantics. Or what do you mean?